src/main/java/copado/onpremise/service/git/Branch.java

```diff
@@ -1,4 +1,12 @@
 package copado.onpremise.service.git;
 
-public interface Branch {
```

Comment:
We think that this interface should not be deleted.
It has been added to do not link the service interface layer into the git library `jgit` that is used in the implementation.
In the future some methods will be added into this interface.

Reply:
If the intend to extend source control system to more than just Git. Then, use marker interface does make sense. I'll need to check a few access modifiers.